front/src/utils/date.ts

```diff
@@ -18,22 +18,29 @@ const userTimeZone = dayjs.tz.guess(); // Format : 'Europe/Paris'
  * @param dateTimeString date string in ISO format
  * @returns string "HH:MM:SS"
  */
-export function extractHHMMSS(dateTimeString?: string) {
+export function extractHHMMSS(dateTimeString?: string): string {
```

Comment:
When you have specific cases you want to handle, you need to write some unit tests that give these specific inputs to the function (like the one you gave in the issue), this helps to document the function behavior.

-12/09/2024 12:00:00 => 12:00
-2/09/2024 07:00:00 => 7:00:


Most importantly, the function is supposed to take _date string in ISO format_, it is not supposed to take `12/09/2024 12:00:00` `2/09/2024 07:00:00` as input.

If we want this function to handle different formats, it needs to be more robust and fully tested

Reply:
if we handle this format you need to use dayjs instead of js `Date` api
![Screenshot_20240904_124630](https://github.com/user-attachments/assets/71d1745c-7462-48ba-bcd6-b576de99afe8)


dayjs('2/09/2024 07:00:00', ['DD/MM/YYYY HH:mm:ss', 'D/MM/YYYY HH:mm:ss'])